include/RAJA/pattern/atomic.hpp

```diff
@@ -80,6 +80,13 @@ namespace RAJA
  */
 
 
+namespace
+{
+  template <typename T>
+  using remove_volatile_t = typename std::remove_volatile< T >::type;
+}
+
+
```

Comment:
Should this be just defined once in a util header?
I thought unnamed namespaces were discouraged in headers.

Reply:
agreed.  Seems like it should go in camp

Reply:
I removed this in my update.